spaces/S000099/README.md

```diff
@@ -6,7 +6,9 @@ refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-Let $X$ be the unit square $[0,1] \times [0,1] \subset \mathbb{R}^2$. If $(x,y)$ is a point off the diagonal, define neighborhoods of $(x,y)$ to be sets of the form $N_\epsilon(x,y) = \{(x,t)\ |\ |t-y| < \epsilon, t \neq x\} $. If $(x,x)$ is a point on the diagonal, define neighborhoods of $(x,x)$, for every finite collection $\{x_i\} \subset [0,1]$ which does not contain $x$, as follows: $\{(t,y) \in X\ |\ |y-x|<\epsilon, t \not\in \{x_0, \dots, x_n\}\}$.
+Let $X$ be the unit square $[0,1] \times [0,1] \subset \mathbb{R}^2$. For a point $(x,y)$ not on the diagonal (for $x \ne y$), neighborhoods are given by the vertical segments $\{x\} \times (y - \varepsilon,y + \varepsilon)$ for $\varepsilon > 0$. For a point $(x,x)$ on the diagonal, open neighborhoods are given by the incomplete horizontal strips $S \times (x - \varepsilon,x + \varepsilon)$ for $\varepsilon > 0$ and $x \notin S$, $[0,1] \setminus S$ finite.
```

Comment:
```suggestion
Let $X$ be the unit square $[0,1] \times [0,1] \subset \mathbb{R}^2$. For a point $(x,y)$ not on the diagonal (for $x \ne y$), basic open neighborhoods are given by the vertical segments $\{x\} \times (y - \varepsilon,y + \varepsilon)$ for $\varepsilon > 0$. For a point $(x,x)$ on the diagonal, basic open neighborhoods are given by open horizontal strips minus a finite number of vertical lines, i.e., sets of the form $S \times (x - \varepsilon,x + \varepsilon)$ for $\varepsilon > 0$ and $S$ cofinite subset of $[0,1]$ not containing $x$.
```
Also needed "basic nbhds", i.e., nbhds belonging to a nbhd base at the point, not just any nbhds.